Sources/YMatterType/Elements/TypographyLabel.swift

```diff
@@ -10,16 +10,42 @@ import UIKit
 
 /// A text label that supports `Typography`.
 /// You can optionally set `maximumPointSize` or `maximumScaleFactor` to set a cap on Dynamic Type scaling.
-open class TypographyLabel: UILabel {
+@IBDesignable open class TypographyLabel: UILabel {
```

Comment:
```suggestion
@IBDesignable
open class TypographyLabel: UILabel {
```
Let's place @IBDesignable on its own line (stylistic choice)